src/CC-SA-1.0.xml

```diff
@@ -0,0 +1,284 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="CC-SA-1.0" name="Creative Common Share Alike 1.0 Generic" listVersionAdded="3.26.0">
+      <crossRefs>
+         <crossRef>https://creativecommons.org/licenses/sa/1.0/legalcode</crossRef>
+      </crossRefs>
+      <text>
+         <titleText>
+            <p>Creative Commons Legal Code</p>
+            <p>ShareAlike 1.0 Generic</p>
```

Comment:
The failing test appears to be due to the word "Generic" appearing in the title text for the version currently on Creative Commons' website at https://creativecommons.org/licenses/sa/1.0/legalcode; but not in the version of the license that appears in [aspell-pl](http://ftp.gnu.org/gnu/aspell/dict/pl/). Other instances of the license out in the wild, such as at https://www.solars.de/cgi-bin/p.pl/www/lizenzen/CC_SA_1_0.html, appear to omit "Generic" as well.

I note also that the version on CC's website doesn't include "Creative Commons Legal Code" at the top, but aspell-pl and the solars.de versions do have that language.

To address this, I'll add a commit which will make these parts optional. From some testing on my machine, looks like `<optional>` tags will work within `<titleText>` tags.

I'll add a subsequent commit